ijson/backends/yajl2_c.py

```diff
@@ -0,0 +1,26 @@
+#
+# Contributed by Rodrigo Tobar <rtobar@icrar.org>
+#
+# ICRAR - International Centre for Radio Astronomy Research
+# (c) UWA - The University of Western Australia, 2016
+# Copyright by UWA (in the framework of the ICRAR)
+# All rights reserved
+#
+'''
+Wrapper for _yajl2 C extension module
+'''
+import decimal
+
+from ijson import common
+from ._yajl2 import basic_parse as _basic_parse  # @UnresolvedImport
+from ._yajl2 import parse as _parse  # @UnresolvedImport
+from ._yajl2 import items as _items  # @UnresolvedImport
```

Comment:
`pyyajl2` was just a subjective preference… But I see your point on consistency with other libraries, so let's leave the naming as is. Thanks for pointing it out!